packages/identity/package.json

```
@@ -2,21 +2,21 @@
     "name": "@semaphore-protocol/identity",
     "version": "4.0.0-alpha.4",
     "description": "A library to create Semaphore identities.",
+    "type": "module",
     "license": "MIT",
-    "main": "dist/index.node.js",
-    "browser": "dist/index.browser.mjs",
+    "main": "dist/index.browser.js",
+    "types": "dist/types/index.d.ts",
     "exports": {
         ".": {
+            "types": "./dist/types/index.d.ts",
             "node": {
-                "import": "./dist/index.node.mjs",
-                "require": "./dist/index.node.js"
+                "import": "./dist/index.node.js",
```

```suggestion
                "default": "./dist/index.node.js",
```

Add the same for the other similar packages.